lib/graphlient/errors/execution_error.rb

```diff
@@ -19,6 +19,12 @@ def to_s
         end.join("\n")
       end
 
+      def responsee
+        warn "The `#responsee' method is deprecated since it has a typo. Please use the `#response' method instead."
+
+        @responsee
+      end
```

Comment:
I think you have meant to add this to `lib/graphlient/errors/graphql_error.rb`.

Reply:
right, thanks. my multitasking isn't going too well here ;)